### DISCUSSION TITLE
Sponsored Brand Report

### DISCUSSION THREAD

**Discussion by Dipikesh** [amzn/ads-advanced-tools-docs#63] [General]
I need sponsored brand report at the level of campaign, adGroup and ad. At the same time i want the report to be of date range, (say last 7 days or last 14 days). 
I tried report API v2, but it needs reportDate:  Each report contains a single day's worth of data. For data over a longer time period, make multiple report requests.

Without making multiple request , is it a way to get sponsored report for a date range,

**Comment by elizafarley** [comment]
Hi @Dipikesh, in version 2 you do need to request each day individually. 

[Version 3 reporting](https://advertising.amazon.com/API/docs/en-us/reporting/v3/overview) does support longer time periods, but for campaign, ad group, and product ad reports, only Sponsored Products is currently supported. We are working on support for Sponsored Brands in version 3 reporting, and will post in the [release notes](https://advertising.amazon.com/API/docs/en-us/info/release-notes) once this functionality is available.
